src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -714,6 +714,25 @@ public void TestEuroCurrencyFromString(object input, string expected, string lan
             Assert.AreEqual(expected, StandardFilters.Currency(context: _contextV20, input: input, languageTag: languageTag));
         }
 
+        [Test]
+        [TestCase(null)]
+        [TestCase("")]
+        public void TestNullOrEmptyInputCurrency(string input)
+        {
+            // Set the thread culture and test for backward compatibility
+            using (CultureHelper.SetCulture("en-US"))
+            {
+                Helper.AssertTemplateResult(
+                    expected: string.Empty,
+                    template: "{{ input | currency: 'de-DE' }}");

```

Comment:
```suggestion
                    expected: input,
                    template: "{{ input | currency: 'de-DE' }}",
                    localVariables: Hash.FromAnonymousObject(new { input = input });
```

Reply:
Fixed the testcase. the expected is always string.Empty for both Null and Empty inputs. just changed that part in the above suggestion

Reply:
Of course, a template render doesn't return null. My mistake!